manifests/plugin/rpm.pp

```diff
@@ -1,8 +1,12 @@
 # @summary Pulp RPM plugin
 # @param use_pulp2_content_route
 #   Whether to redirect the legacy (Pulp 2) URLs to the content server
+#
+# @param keep_changelog_limit
+#   Pulpcore's KEEP_CHANGELOG_LIMIT setting. Uses Pulpcore's default when undefined. Increasing this limit will cause pulpcore workers to use more memory when more changelogs are available in the repo metadata.
```

Comment:
I usually limit the line length, like this:
https://github.com/theforeman/puppet-pulpcore/blob/d347abdac8091ed725a532427c3aa380a63bcbc9/manifests/init.pp#L30-L33

I use vim to do that (shift v -> gq wraps).

Reply:
Hugely useful. Thanks!